openmetadata-ui/src/main/resources/ui/src/components/DataAssets/DataAssetsHeader/DataAssetsHeader.component.tsx

```diff
@@ -92,22 +92,30 @@ export const ExtraInfoLabel = ({
   label,
   value,
   dataTestId,
+  isNewDesign = false,
+  showAsATag = false,
 }: {
   label: string;
   value: string | number;
   dataTestId?: string;
+  isNewDesign?: boolean;
```

Comment:
We don't need this. 
This is used at one place only and we want to redesign, so remove this param